source/type/type.h

```diff
@@ -614,12 +614,14 @@ struct Line : public Header, Nameable, HasMessages, Codes{
     MultiLineString shape;
     boost::optional<boost::posix_time::time_duration> opening_time, closing_time;
 
+    std::map<std::string,std::string> properties;
```

Comment:
As I explain yesterday in meeting with Stephan Simart, I just need it on lines for the moment.
So I just code what I use (lazy programming technique).
But My implementation could be completed by someone when the need comes.
If you think that a full implementation is mandatory please tell me and I will code it.
Please ask Stephan and tell me your decision.